cmd/loadbalancer/main.go

```diff
@@ -47,8 +47,8 @@ func main() {
 
 	// Register standard Go metric collectors, which are by default registered when using global registry.
 	reg.MustRegister(
-		version.NewCollector("observable-demo"),
-		prometheus.NewBuildInfoCollector(),
+		version.NewCollector(""),
```

Comment:
I like the idea of doing `build_info` only so you can have nice version dashboard finally (:

Or use just `NewBuildInfoCollector` for it (: